### PR TITLE
Properly qualify `rand` and `logpdf`

### PR DESCRIPTION
**Pull request by cscherrer**

### CODE REVIEW

**Review by DilumAluthge — DISMISSED** [review]

**Review by millerjoey — APPROVED** [review]

File: src/distributions/for.jl [hunk 1]
```diff
@@ -89,14 +89,15 @@ function For(f::F, θ::T) where {F,T<:Base.Generator}
...
-@inline function logpdf(d::For{F,T}, x) where {F,T<:Base.Generator}
+@inline function Distributions.logpdf(d::For{F,T}, x) where {F,T<:Base.Generator}
     s = 0.0
     for (θj, xj) in zip(d.θ, x)
         s += logpdf(d.f(θj), xj)
     end
     return s
 end
 
+
```
Comment (cscherrer) [inline src/distributions/for.jl:100]:
```suggestion
```

**Comment by cscherrer** [comment]
@DilumAluthge @millerjoey it's says I dismissed your reviews, which.... why? Not sure. I think this is good to go, any concerns?

**Review by DilumAluthge — APPROVED** [review]
LGTM.

**Comment by DilumAluthge** [comment]
I think GitHub auto-dismisses "stale" reviews when new commits are pushed.

**Comment by millerjoey** [comment]
No problem. All good for me

**Pull request merged**